src/services/user.service.js

```diff
@@ -0,0 +1,36 @@
+const { User } = require('../models/User.model');
+const { generateRandomId } = require('../Random');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const getById = async (id) => {
+  return User.findByPk(+id);
```

Comment:
Ensure that the `id` parameter is a valid number before converting it with `+id`. Consider adding validation to handle cases where `id` might not be a number.